38c3.html

```diff
@@ -107,42 +107,42 @@ <h2>Delta Chat @ 38c3</h2>
         alt="deltachat logo"
       />
       <p>
-        Delta Chat is a decentralized and secure messenger app
+        Delta Chat is a secure and decentralized super app offering
```

Comment:
i'd add an ellipsis here as it is not obvious by typography that the sentence continues

```suggestion
        Delta Chat is a secure and decentralized super app offering ...
```